Utilities/Processes.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Diagnostics;
+
+namespace Utilities
+{
+    public class Processes
+    {
+        private const string eliteDangerousProcessName = "EliteDangerous";
```

Comment:
I *think* this also works for beta but sadly we cannot test.

Reply:
I don't think they would change the process name for beta builds, but even it they did it would probably be to add "beta" as a prefix of suffix. The `IsProcessRunning()` method looks for processes that contain "EliteDangerous" and would consequently accept "EliteDangerousBeta", "BetaEliteDangerous", or "EliteDangerous_Beta" as valid.